org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/DOMToModelPopulator.java

```diff
@@ -945,6 +893,20 @@ public IMemberValuePair[] getMemberValuePairs() {
 		};
 	}
 
+	public static LocalVariable toLocalVariable(VariableDeclarationFragment fragment, JavaElement parent) {
```

Comment:
Are there other locations in this file where this method could be used (to showcase that this helper method is in the right class)? Otherwise, we might better move it to the `javac` bundle.

Reply:
I don't think it can be used anywhere else, since in ECJ the `LocalVariable` are created on demand in the binding class instead of in the usual class that builds the model. The main reason I put it here is that it uses protected methods that are in the class, and it's signature is very similar to `toLocalVariable` that accepts a `SingleVariableDeclaration` instead of the `VariableDeclarationFragment`. I can move this method into `JavacVariableBinding` if you think it makes more sense.

Reply:
OK, thanks for the details. I think it would indeed be better to move this method to JavacVariableBinding.